src/ExpoPushNotificationsServiceProvider.php

```diff
@@ -16,7 +16,7 @@ class ExpoPushNotificationsServiceProvider extends ServiceProvider
      *
      * @return void
      */
-    public function boot()
+    public function boot(\Illuminate\Routing\Router $router)
```

Comment:
I think we don't need the `$router` to be injected we can retrieve it from `$this->app['router']`

Reply:
Done.